tools/extract_assets.py

```diff
@@ -152,7 +152,8 @@ def main():
         for currentPath, _, files in os.walk(os.path.join("assets", "xml")):
             for file in files:
                 fullPath = os.path.join(currentPath, file)
-                if file.endswith(".xml"):
+                # ZAPD can't handle audio, skip those XMLs.
+                if file.endswith(".xml") and (fullPath.find("audio") == -1):
```

Comment:
Just checked and we do indeed were passing audio xmls to ZAPD.
I wonder why it just complained now, there doesn't seem to be any change that to the format of the xmls to guarantee this problem.
Also, the error it gives doesn't make much sense. I wonder if that check if bugged.